specifyweb/urls.py

```diff
@@ -52,4 +53,5 @@
     url(r'^export/', include(export_urls)), # permissions added
     url(r'^permissions/', include(permissions_urls)), # permissions added
     # url(r'^testcontext/', include()),
+    url(r'^statistics/', include(stat_urls))
```

Comment:
Need to add back-end permission for the stats page. Research how other endpoints did that and do the same

Reply:
Added per table basis